src/main/java/com/minecolonies/coremod/util/AchievementUtils.java

```diff
@@ -37,7 +37,7 @@ public static void syncAchievements(@NotNull final Colony colony)
 
         if (colony.getWorld() != null)
         {
-            for (final Advancement achievement : colony.getAchievements())
+            for (final Advancement achievement : colony.getStatsManager().getAchievements())
             {
                 //todo
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)